postgres/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM mdillon/postgis
 
 COPY initdb.sql docker-entrypoint-initdb.d/
-COPY serviceAccountKey.json serviceAccountKey.json
+#COPY serviceAccountKey.json serviceAccountKey.json
```

Comment:
Service Account Key should be copied into docker container. It is needed by wal-g to backup into google cloud.